storage/imx6ull-flash/imx6ull-flash-test.c

```diff
@@ -318,6 +320,70 @@ void test_bootrom(void)
 }
 
 
+static unsigned int flashdrv_checkErased(const void *buff, size_t boffs, size_t blen)
```

Comment:
Copy-pasted code in test might backfire badly if you update one of copies without updating the other

Reply:
Yeah, I agree. The problem is that the ECC checking code in `flashdev.c` is specific for `flashsrv` (e.g. uses `storage_t`) and `imx6ull-flash-test.c` uses only `flashdrv` interface so it has to be adapted. Maybe just leave a comment for now?

Reply:
Ok